app/code/community/Bolt/Boltpay/controllers/ApiController.php

```diff
@@ -185,12 +185,6 @@ protected function getCaptureAmount($transaction) {
             return $transaction->capture->amount->amount/100;
```

Comment:
should we check existence of `$transaction->capture` and `$transaction->capture->amount`?

Reply:
I think we only care about the amount, right? isset() will return true only if all variables in the chain are defined I believe. So if $transaction->capture is undefined, isset($transaction->capture->amount->amount) will return false and won't error out. Is that what you're asking?

Reply:
aha - PHP is such a generous language. I was worried about potential nullpointerexception kind of error but it sounds like it won't happen. thanks for clarification.